dotnet/src/Autogen.Ollama/DTOs/CompleteChatMessage.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// CompleteChatMessage.cs
+
+using System.Text.Json.Serialization;
+
+namespace Autogen.Ollama;
+
+public class CompleteChatMessage : ChatMessage
```

Comment:
Is that the type for chat response?

Reply:
Yes

Reply:
Maybe rename it to `ChatResponse` so as to match the name from ollama-js and ollama-python libraries?

https://github.com/ollama/ollama-js/blob/63dcfbb44d3b22d9a1ae4bb333f158dbd5a119d7/src/interfaces.ts#L135C18-L135C31

And it shouldn't inherit from `ChatMessage` I guess?

Reply:
done

Reply:
fixed previous requests.. let me check this to implement it correctly